armadillo/tests/test_pi_stacking-individual.py

```diff
@@ -0,0 +1,19 @@
+# Import package, test suite, and other packages as needed
+import armadillo
+import pytest
+import sys
+import MDAnalysis as mda
+
+
+def test_armadillo_imported():
+    """Sample test, will always pass so long as import statement worked"""
+    assert "armadillo" in sys.modules
+
+u = mda.Universe('g3.pdb', guess_bonds=True)
+
+ring_list = armadillo.find_all_rings(u)
```

Comment:
if you had a single `test.py` file, you could perhaps load the test `Universe` once, then pass it to all test functions.